versions/2016-10-31.md

```diff
@@ -780,7 +780,7 @@ NOTE: To specify a Cognito UserPool as an event source for a Lambda function, bo
 ```yaml
 Type: Cognito
 Properties:
-  Bucket: Ref: MyUserPool
```

Comment:
Looks like this is fixed in #1233 PR and merged into `develop`. Could you please pull and rebase the latest changes to see the change fixed?